.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish NuGet Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '7.0.x'  # Replace with the desired version of .NET SDK
```

Comment:
what version(s) should we support

Reply:
8.0 will have long term support, so we can switch to 8.8 when it is official.